docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -179,10 +209,14 @@ The following command creates [eas.json](/build/eas-json.mdx) in the project's r
 
 #### Configure EAS Build
 
-There are three more steps to configure EAS Build for running E2E tests as part of the build:
+There are few more steps to configure EAS Build for running E2E tests as part of the build:
 
-- Detox tests are run in the iOS simulator. We will define a build profile that builds your app for simulator.
-- Install the [`applesimutils`](https://github.com/wix/AppleSimulatorUtils) command line util required by Detox.
+- Android tests:
+  - Tests are run in the Android Emulator. You will define a build profile that builds your app for the emulator (produces an `apk` file).
```

Comment:
```suggestion
  - Tests are run in the Android emulator. You will define a build profile that builds your app for the emulator (produces an `apk` file).
```
A quick scan shows we tend to prefer lowercase 'emulator'.

Reply:
https://github.com/expo/expo/actions/runs/3219482687/jobs/5264910090